docs/.vitepress/config.ts

```diff
@@ -248,51 +247,60 @@ export default defineConfig({
 
     // https://vitepress.dev/reference/default-theme-config
     nav: [
-      { text: 'Guide', link: '/guide/getting-started.md' },
-      { text: 'Config', link: CONFIG_LINK },
-      { text: 'Plugins', link: '/builtin-plugins/' },
```

Comment:
As a note (also mentioned in the call)

We lose the links to existing built-in plugins (e.g. https://rolldown.rs/builtin-plugins/esm-external-require) right now.

https://deploy-preview-7092--rolldown-rs.netlify.app/apis/plugin-api#builtin-plugins is only covering the overview of built-in plugins. Linking them there should be enough for now.
It also works to put the whole "plugins" section in the side bar of the "guides" section.